Code.Rmd

```diff
@@ -173,25 +412,27 @@ For now, you should always use `.onLoad()` unless explicitly directed otherwise.
 
 Some common uses of `.onLoad()` and `.onAttach()` are:
 
-*   To display an informative message when the package loads. This might make 
-    usage conditions clear, or display useful tips. Startup messages is one 
-    place where you should use `.onAttach()` instead of `.onLoad()`. To display 
-    startup messages, always use `packageStartupMessage()`, and not `message()`. 
-    (This allows `suppressPackageStartupMessages()` to selectively suppress 
-    package startup messages).
+* To display an informative message when the package loads. This might make 
+  usage conditions clear, or display useful tips. A startup messages is one 
+  place where you should use `.onAttach()` instead of `.onLoad()`. To display 
+  a startup message, always use `packageStartupMessage()`, and not `message()`. 
+  This allows `suppressPackageStartupMessages()` to selectively suppress 
+  package startup messages.
 
     ```{r, eval = FALSE}
     .onAttach <- function(libname, pkgname) {
       packageStartupMessage("Welcome to my package")
     }
     ```
     
-*   To set custom options for your package with `options()`. To avoid conflicts
-    with other packages, ensure that you prefix option names with the name
-    of your package. Also be careful not to override options that the user
-    has already set.
+* To set custom options for your package with `options()`. To avoid conflicts
```

Comment:
Do we still believe in this? Or should you define helpers to get options that you can document, and that provide defaults?